CHANGELOG.next.asciidoc

```diff
@@ -325,6 +325,9 @@ otherwise no tag is added. {issue}42208[42208] {pull}42403[42403]
 - Replace Ubuntu 20.04 with 24.04 for Docker base images {issue}40743[40743] {pull}40942[40942]
 - Publish cloud.availability_zone by add_cloud_metadata processor in azure environments {issue}42601[42601] {pull}43618[43618]
 - Disable sending telemetry to Microsoft when using their Go fork for producing FIPS-capable artifacts {pull}44865[44865]
+- The `add_cloud_metadata` processor does not support the Azure provider in FIPS-capable artifacts {pull}xxxx[xxxx]
```

Comment:
I think we can delete the first line

Reply:
Actually, it turns out we're not supposed to have any FIPS-related changelog entries (see https://github.com/elastic/beats/pull/43428#discussion_r2012737373), so I've deleted all three lines.